zulipterminal/ui_tools/messages.py

```diff
@@ -632,6 +632,45 @@ def soup2markup(
 
                 source_text = f"Original text was {tag_text.strip()}"
                 metadata["time_mentions"].append((time_string, source_text))
+            elif tag == "div" and "spoiler-block" in tag_classes:
+                # SPOILERS
+                header = element.find(class_="spoiler-header")
+                header.contents = [part for part in header.contents if part != "\n"]
+
+                if not header.contents:
+                    default = BeautifulSoup("<p>Spoiler</p>", "html.parser")
+                    header.contents.append(default)
+
+                processed_header = cls.soup2markup(header, metadata)[0]
+
+                processed_header_text = "".join(
+                    part[1] if isinstance(part, tuple) else part
+                    for part in processed_header
+                )
+
+                # Limit to the first 10 characters and append "..."
+                if len(processed_header_text) > 10:
+                    processed_header_text = processed_header_text[:10] + "..."
```

Comment:
If you have a literal like 10, it's almost always useful to give it a name, particularly when it's referenced in multiple locations like here. Other than being descriptive, it ensures everything gets updated at the same time, which reduces one bug location.